--- python-efl-1.13.0/setup.py~	2015-02-07 10:15:05.000000000 +0000
+++ python-efl-1.13.0/setup.py	2015-04-28 02:05:23.343148012 +0100
@@ -372,7 +372,7 @@
                             ["efl/dbus_mainloop/dbus_mainloop" + module_suffix,
                             "efl/dbus_mainloop/e_dbus.c"],
                             extra_compile_args=list(set(dbus_cflags + ecore_cflags)),
-                            extra_link_args=dbus_libs + ecore_libs)
+                            extra_link_args=dbus_libs + ecore_libs + eina_libs)
     ext_modules.append(dbus_ml_ext)
 
     # === Elementary ===
