diff -up python-efl-1.26.1/setup.py.1~ python-efl-1.26.1/setup.py
--- python-efl-1.26.1/setup.py.1~	2023-11-28 01:46:20.813977067 +0100
+++ python-efl-1.26.1/setup.py	2023-11-28 01:46:51.140116116 +0100
@@ -114,7 +114,6 @@ ext_modules = []
 py_modules = []
 packages = ['efl']
 common_cflags = [
-    '-fno-var-tracking-assignments',  # seems to lower the mem used during build
     '-Wno-misleading-indentation',  # not needed (we don't indent the C code)
     '-Wno-deprecated-declarations',  # we bind deprecated functions
     '-Wno-unused-variable',  # eo_instance_from_object() is unused
