diff -uraN python-efl-1.25.0/setup.py omv-python-efl-1.25.0/setup.py
--- python-efl-1.25.0/setup.py	2020-06-25 18:46:38.312786000 +0200
+++ omv-python-efl-1.25.0/setup.py	2020-10-10 20:00:09.661935479 +0200
@@ -211,17 +211,12 @@
 py_modules = []
 packages = ["efl"]
 common_cflags = [
-    "-fno-var-tracking-assignments",  # seems to lower the mem used during build
     "-Wno-misleading-indentation",  # not needed (we don't indent the C code)
     "-Wno-deprecated-declarations",  # we bind deprecated functions
     "-Wno-unused-variable",  # eo_instance_from_object() is unused
     "-Wno-format-security",  # some cc don't like the way cython use EINA_LOG macros
     # "-Werror", "-Wfatal-errors"  # use this to stop build on first warnings
 ]
-# remove clang unknown flags
-if os.getenv("CC") == "clang":
-    common_cflags.remove('-fno-var-tracking-assignments')
-
 
 if set(("build", "build_ext", "install", "bdist", "sdist")) & set(sys.argv):
     sys.stdout.write("Python-EFL: %s\n" % RELEASE)
