diff --git a/setup.py b/setup.py
index 82dfe26..d41030b 100755
--- a/setup.py
+++ b/setup.py
@@ -211,7 +211,6 @@ ext_modules = []
 py_modules = []
 packages = ["efl"]
 common_cflags = [
-    "-fno-var-tracking-assignments", # seems to lower the mem used during build
     "-Wno-misleading-indentation", # not needed (we don't indent the C code)
     "-Wno-deprecated-declarations", # we bind deprecated functions
     "-Wno-unused-variable", # eo_instance_from_object() is unused
