diff --git a/setup.py b/setup.py
index 82dfe26..cc9401c 100755
--- a/setup.py
+++ b/setup.py
@@ -435,7 +434,7 @@ if set(("build", "build_ext", "install", "bdist", "sdist")) & set(sys.argv):
                             extra_compile_args=list(set(dbus_cflags +
                                                         ecore_cflags +
                                                         common_cflags)),
-                            extra_link_args=dbus_libs + ecore_libs)
+                            extra_link_args=dbus_libs + ecore_libs + eina_libs)
     ext_modules.append(dbus_ml_ext)
 
     # === Elementary ===
